README.md

```diff
@@ -549,6 +560,22 @@ This framework helps users define rules to capture scenarios like criteria for s
 
 For details about how you can use set a custom stopping criteria and perform custom operations, see [examples/trainercontroller_configs/Readme.md](examples/trainercontroller_configs/Readme.md)
 
+
+## Experiment Tracking
+
+Experiment tracking in fms-hf-tuning allows users to track their experiments with known trackers like [Aim](https://aimstack.io/) or custom trackers built into the code like
+[FileLoggingTracker](./tuning/trackers/filelogging_tracker.py)
+
+The code supports currently two trackers out of the box, 
+* FileLoggingTracker : A built in tracker which supports logging training loss to a file.
+* [Aimstack](https://aimstack.io/) : A popular opensource tracker which can be used to track any metrics or metadata from the experiments.
+
+    To use this, you need to first install the package marked as an optional dependency, as described in [Installation](#installation).
```

Comment:
Do you think it is worthwhile to repeat the installation instructions again in the tracking specific doc to make it complete and avoid the reference loop :)

Reply:
sure.